sycl/doc/syclgraph/SYCLGraphUsageGuide.md

```diff
@@ -453,6 +453,113 @@ node kernelNode = myGraph.add([&](handler& cgh) {
 dynParamAccessor.update(bufferB.get_access());
 ```
 
+### Dynamic Command Groups
+
+Example showing how a graph with a dynamic command group node can be updated.
+
+```cpp
+queue Queue{};
+exp_ext::command_graph Graph{Queue.get_context(), Queue.get_device()};
```

Comment:
We should include the using statement in the example code if we're using it. Also the other examples seem to favor `sycl_ext`, would be good to be consistent.